packages/cinterop/src/nativeDarwin/kotlin/io/realm/kotlin/internal/interop/ErrorCode.kt

```diff
@@ -120,9 +120,9 @@ actual enum class ErrorCode(
     RLM_ERR_CUSTOM_ERROR("CustomError", realm_errno.RLM_ERR_CUSTOM_ERROR),
     RLM_ERR_CLIENT_USER_NOT_FOUND("ClientUserNotFound", realm_errno.RLM_ERR_CLIENT_USER_NOT_FOUND),
     RLM_ERR_CLIENT_USER_NOT_LOGGED_IN("ClientUserNotLoggedIn", realm_errno.RLM_ERR_CLIENT_USER_NOT_LOGGED_IN),
```

Comment:
`RLM_ERR_CLIENT_USER_NOT_LOGGED_IN` Shouldn't be removed?

Reply:
I guess not. I must have missed something as `RLM_ERR_CLIENT_USER_NOT_LOGGED_IN` is not remove from the `error_codes.h`. I will readd them for common and JVM too.